code/area.dm

```diff
@@ -35,8 +35,8 @@ ABSTRACT_TYPE(/area) // don't instantiate this directly dummies, use /area/space
 	var/tmp/sims_score = 100
 	var/virtual = 0
 
-	/// Whether or not the tile will be drawn on the station map.
-	var/render_on_map = FALSE
+	/// A flag system for determining which minimap types this area should be rendered on.
```

Comment:
since we don't have a default value, listing the flag pattern here is worthwhile e.g. `MAP_*` in `minimap.dm`